website/payments/views.py

```diff
@@ -145,14 +146,21 @@ def get_queryset(self) -> QuerySet:
 
     def get_context_data(self, *args, **kwargs):
         filters = []
-        for i in range(13):
+
+        for i in range(85):
```

Comment:
85 is kind of a weird number?... Is it based on the legal storage period for payment info? For the user it might be nice if years in which there were no payments (or at least years before their first payment) are not shown.